bin/cron/applab_datasets/covid19

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env ruby
+
+# This script fetches covid19 data and uploads to the shared firebase channel.
+require_relative '../../../deployment'
+require 'cdo/only_one'
+require 'net/http'
+require 'csv'
+require 'date'
+require 'datapackage'
+require_relative '../../../shared/middleware/helpers/firebase_helper'
+
+US_DATA_URL = "https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv"
+MAX_DAYS = 90 # there are 55 states/territories, and the table must be less than 5000 total rows (5000/55 = 90.9)
+MAX_WEEKS = 26 # there are 188 countries, and the table must be less than 5000 total rows (5000/188 = 26.5)
+
+def get_us_covid19_data
+  response = Net::HTTP.get_response(URI(US_DATA_URL))
+  return unless response.code == '200'
+  response.body.force_encoding('utf-8')
+  csv = CSV.new(response.body)
```

Comment:
[this breaks the convention for past live dataset scripts, but might be useful to call out anyways]

we can parse this CSV to a `CSV::Table`, which means we can reference row values by their column name, rather than their index: https://ruby-doc.org/stdlib-2.6.1/libdoc/csv/rdoc/CSV.html#class-CSV-label-CSV+with+headers

```ruby
data = CSV.parse(response.body, headers: true)
data.first.field(:date)
  => '2020-01-21'
```